### DISCUSSION TITLE
FEATURE REQUEST: Username, Password, and OTP/2FA Fields Should Be Able To Be Autopopulated With Password Managers

### DISCUSSION THREAD

**Discussion by johnwyles** [home-assistant/frontend#10829] [Accessibility]
I haven't tried LastPass but with 1Password the Username, Password, and OTP fields are not about to auto-populate and it is a nightmare when you have setup things like internal and external endpoints and change between them and/or have setup reverse proxying.

I am not a front-end guy myself so I am not sure how these password managers clue in on the fields but I would imagine is a class name in the HTML or field name. It seems like a very simple fix and one that would not take long at all for someone more knowledgeable about this stuff than I am to implement.

**Comment by spacegaier** [comment]
Unfortunately, it does not seem to be that easy...
https://github.com/home-assistant/frontend/issues/10274 
https://github.com/home-assistant/frontend/issues/3133 
https://github.com/home-assistant/frontend/issues/8046
